edudoc/judge-tutorial/judge-tutorial-hy.md

```diff
@@ -78,4 +78,4 @@
 
 <style>
   :root { font-size: 0.85em; }
-</style>
\ No newline at end of file
+</style>
```

Comment:
What happened here?

Reply:
I don't know. The diff in my editor had a new blank line at the end.